SofieC93.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: SofieC93
+pageUrl: SofieC93.github.io
+timestamp: 2017-02-27
+---
```

Comment:
Hi @SofieC93! It looks like there needs to be a line break after the characters on line five. Let me know if you need any help!